.github/workflows/r.yml

```diff
@@ -248,6 +248,7 @@ jobs:
     env:
       ARROW_R_CXXFLAGS: "-Werror"
       _R_CHECK_TESTS_NLINES_: 0
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
```

Comment:
Now you're using a different method to the one I started, is this line still needed?